actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -35,7 +35,9 @@ module ImplicitRender
     include BasicImplicitRender
 
     def default_render
-      if template_exists?(action_name.to_s, _prefixes, variants: request.variant)
+      if Object.const_defined?(const_name = renderable_name(action_name.to_s.underscore, _prefixes))
+        render(const_name.constantize.new)
```

Comment:
I'm sorry if I'm missing something. How would you pass initialization arguments to a renderable?

Reply:
@MatheusRich you aren't missing something. In this implementation, arguments are not passed to a renderable when it is rendered implicitly. How do you feel about @bradgessler's proposed solution (with prior working art) to assign ivars to renderables?

Reply:
I like the idea of having a method that automatically assigns ivars but can be overridden.